### ISSUE TITLE
First preamble paragraph does not have the implicit `lead` role in the preview

### ISSUE THREAD

**Issue by Vampire** [asciidoctor/asciidoctor-intellij-plugin#1567]
## Observed vs. expected behavior

https://docs.asciidoctor.org/asciidoc/latest/blocks/preamble-and-lead/#preamble-style says that with the default stylesheet the first preamble paragraph gets the implicit `lead` role if no explicit one is given.

If you generate PDF or HTML from the toolbar this is also done.

But the in-editor preview does not apply this implicit `lead` role for the first preamble paragraph.

While the "with the default stylesheet" is of course an escape-hatch, it would probably be more consistent to also there use the implicit `lead` role.

## Steps to reproduce

- Have a preamble
- Look at the preview

## Environment

Plugin Version: 0.41.8

IntelliJ Details: IntelliJ IDEA 2023.3.4 (Ultimate Edition)
Build #IU-233.14475.28, built on February 13, 2024

**Comment by ahus1** [comment]
Thanks for creating this issue. I found the cause, and ask the Asciidoctor project for an update in their stylesheet: https://github.com/asciidoctor/asciidoctor/issues/4564

In the meantime, I'll include a workaround in the plugin, which will be part of the next release. I'll update the issue once a pre-release is available.

**Comment by ahus1** [comment]
Thanks to the Dan's description, I was able to tune the CSS adjustment that it behaves as expected. 

```
= The Intrepid Chronicles

This is the preamble.

What are we going to do?

== Certain Peril

Daylight ...
```

renders with slightly larger preamble

![image](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/3957921/054c5661-6328-4755-9506-670901faea38)

while 

```
= The Intrepid Chronicles

[.anything]
This is the preamble.

What are we going to do?

== Certain Peril

Daylight ...
```

renders with standard font preamble

![image](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/3957921/d625fdb2-4f46-47b7-8730-e2a373ba53e4)

while 

```
= The Intrepid Chronicles

[.anything.lead]
This is the preamble.

What are we going to do?

== Certain Peril

Daylight ...
```

renders with slightly larger preamble

![image](https://github.com/asciidoctor/asciidoctor-intellij-plugin/assets/3957921/46dd0628-98aa-4635-aac5-6aad8a8407f0)

**Comment by ahus1** [comment]
This fix is now available as pre-release 0.41.10.

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
